include/control_toolbox/pid.hpp

```diff
@@ -117,7 +115,6 @@ class Pid
    * \param i_max The max integral windup.
    * \param i_min The min integral windup.
    *
-   * \throws An std::invalid_argument exception is thrown if i_min > i_max
```

Comment:
Don't we want to throw this anymore?

Reply:
we never threw here! And there is a test that set_gains is called with wrong parameter and it should not throw.